build/tools/httpbin.sh

```diff
@@ -55,6 +55,7 @@ httpbin_launch() {
     # it may not support Python version that we actually have (this one
     # still works with 3.4, 20.0.1 is the last one to support 3.5).
     python3 -m pip install --user --upgrade pip==19.1.1
+    python3 -m pip install wheel --user
```

Comment:
Can we do it like this
```suggestion
    python3 -m pip install --user wheel
```
just for consistency? If yes, I'll just change this when applying.

Reply:
Yeah, it was bothering my OCD after I looked at it.  :-)

Reply:
Good to see that I'm not the only one.

But the important thing is that it did fix the CMake build, thanks a lot! Will merge soon.